config/routes.rb

```diff
@@ -4,7 +4,7 @@
   root 'sessions#new'
   get 'login' => 'sessions#new'
   post 'login' => 'sessions#create'
-  delete 'logout' => 'sessions#destroy'
+  get 'logout' => 'sessions#destroy'
```

Comment:
unless there's a really good reason, "destructive" or actions with side effects (like logout) should be accessed via a `post` or some action other than `get`.  strongly suggest modifying the route and UI button accordingly.